packages/web/components/Guild/GuildHero.tsx

```diff
@@ -1,12 +1,50 @@
-import { Avatar, Box, MetaButton, Text, VStack } from '@metafam/ds';
+import {
+  Avatar,
+  Box,
+  EditIcon,
+  IconButton,
+  Link,
+  MetaButton,
+  Text,
+  VStack,
+} from '@metafam/ds';
 import { ProfileSection } from 'components/Profile/ProfileSection';
 import { GuildFragment } from 'graphql/autogen/types';
 import React from 'react';
 
-type Props = { guild: GuildFragment };
+type Props = {
+  guild: GuildFragment;
+  canEdit: boolean;
```

Comment:
@dysbulic on his PR has changed all occurences of `canEdit` with `editing`. So maybe good to rename this as well.